code/go/0chain.net/smartcontract/dbs/event/allocation_blobber_term.go

```diff
@@ -0,0 +1,87 @@
+package event
+
+import (
+	"gorm.io/gorm/clause"
+	"time"
+
+	"gorm.io/gorm"
+)
+
+type AllocationBlobberTerm struct {
+	gorm.Model
+	AllocationID            string        `json:"allocation_id" gorm:"uniqueIndex:idx_alloc_blob; not null"` // Foreign Key
```

Comment:
better add priority for index, to be defined

Reply:
added priority for index. allocation_id (1) has higher priority than blobber_id (2). ptal.

Reply:
higher is 1